cmd/app/flags.go

```diff
@@ -68,7 +68,7 @@ func configureFlags(command *cobra.Command) {
 		"When building a Hugo-compliant documentation bundle, files with filename matching one form this list (in that order) will be renamed to _index.md. Only useful with --hugo=true")
 	_ = vip.BindPFlag("hugo-section-files", command.Flags().Lookup("hugo-section-files"))
 
-	command.Flags().StringSlice("content-files-formats", []string{".md"},
+	command.Flags().StringSlice("content-files-formats", []string{".md", ".html"},
```

Comment:
Why do we need this? This is the default variable values if the flag is not set and shouldn't be related to `document_worker.go`